Dashboard/app/js/lib/views/reports/export-reports-views.js

```diff
@@ -147,7 +175,21 @@ FLOW.ExportReportsAppletView = FLOW.View.extend({
   showGoogleEarthButton: false,
   reportFromDate: undefined,
   reportToDate: undefined,
-  
+  dateRangeSelected: true,
+  exportOption: "range",
+  dateRangeText: Ember.String.loc('_collection_period'),
+  onlyRecentText: Ember.String.loc('_only_recent_submissions'),
+
+  dateRangeSelectedObserver: function () {
+    if (this.get("exportOption") == "range") {
+      this.set('dateRangeSelected', true);
+    } else {
+      this.set('dateRangeSelected', false);
+      FLOW.dateControl.set('fromDate', null);
```

Comment:
We previously made code changes to stop using the `FLOW.dateControl` and instead use `reportFromDate` and `reportToDate` properties.